src/NHibernate/Multi/IMultiAnyQueryBatch.cs

```diff
@@ -0,0 +1,24 @@
+namespace NHibernate
+{
+	/// <summary>
+	/// Universal query batcher
+	/// </summary>
+	public partial interface IMultiAnyQueryBatch
```

Comment:
Naming should be simplified IMO, `IAnyQueryBatch` should be enough. Well, even `IQueryBatch` should be enough indeed, provided the previous batchers are obsoleted.